hieradata/clients/bounce.yaml

```diff
@@ -5,4 +5,8 @@ accounts:
     ssh_keys:
       default:
         key: AAAAB3NzaC1yc2EAAAABIwAAAQEAqUJsLjvxcFROCirP0Nya3UjXuVoS2LuMGqX0Bphq9DWy3vw5Y9sCIRDO3UKqwUgI3wM9AFLpYPTwdTWnerXZsoAzzQIFKcNjYVSiiWCguocK1/oTiwhF2G+XdJ4qbvPauRJg7RrYMKdowGW8eSuJf9iL15i+nA2H2JUhAgCqAf8X2iziEhfoir0kRIwXupjNLhVM9caX0z37ydwbWCX856aexR2bnGYgjY23SPTehB6fa0JugJXgK05yBaqaxj/C2/MuF0gouaog8Kl2s4QBDEoID7zPMvNTdZLDMTQgnHWX49Ed8AsLZNYPhWCQwQr90YNc7c5bGwHwUkZgiAXJCQ==
+  slide_o_mix:
```

Comment:
Just to confirm that this is okay, this is what the username on the system will be. You can change this to whatever is convenient for you.

Reply:
Ah, I was thinking it had to be the LDAP username for the Jenkins infra. Let me fix it real quick.